src/Middleware/SpaServices.Extensions/src/AngularCli/AngularCliBuilder.cs

```diff
@@ -49,32 +50,33 @@ public async Task Build(ISpaBuilder spaBuilder)
             var logger = LoggerFinder.GetOrCreateLogger(
                 spaBuilder.ApplicationBuilder,
                 nameof(AngularCliBuilder));
-            var npmScriptRunner = new NpmScriptRunner(
+            var scriptRunner = new NodeScriptRunner(
                 sourcePath,
-                _npmScriptName,
+                _scriptName,
                 "--watch",
```

Comment:
Are these options valid on all package managers? What if it's called something else in yarn?

Reply:
[According to the Angular docs](https://angular.io/cli/build), the `--watch` param will run the `_scriptName` when files change. This is a parameter for the Angular CLI itself and is independent of whether you used yarn or npm to invoke it.